variants.ini

```diff
@@ -8,7 +8,7 @@ capturesToHand = true
 startFen = rnbqkbnr/pppppppp/8/8/8/8/PPPPPPPP/RNBQKBNR w KQkq - 0 1
 mustCapture = true
 
-[antiatomic]
+[anti_atomic:giveaway]
```

Comment:
This neither follows giveaway nor atomic.

Reply:
https://github.com/ianfab/Fairy-Stockfish/blob/c96acb864bd52fb440a0ae280010d1259e40dfb2/src/variants.ini#L268-L270

Reply:
When there is a variant existing in fairy sf why take the effort to do extra unneeded stuff

Reply:
if possible send a pr please as said in https://github.com/RaviharaV-bot/Liantichess/pull/9#issuecomment-1019037382